src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/EnergyDependentWattSpectrum/Builder/src/construct.hpp

```diff
@@ -0,0 +1,19 @@
+EnergyDependentWattSpectrum construct(){
+  if( not aTabulated_ ){
+    Log::info( "aTabulated not built" );
+  }
+  if( not bTabulated_ ){
+    Log::info( "bTabulated not built" );
+  }
+  try{
+    return EnergyDependentWattSpectrum{ 
+      std::move( aTabulated_.value() ),
+      std::move( bTabulated_.value() ),
+      restrictionEnergy_.value()
+
+    };
+  } catch( std::bad_optional_access& b ){
+    Log::error( "Part of Energy Dependent Watt Spectrum not yet built." );
+    throw;
+  }
```

Comment:
You explicitly test every component while in other cases you just use a single try ... catch ...

Reply:
I think this is a remnant of some debugging I did while creating this. It's been removed.